models/docs.md

```diff
@@ -47,7 +47,15 @@ The bid match type associated with this record; values include 'Broad', 'Exact',
 {% enddocs %}
 
 {% docs budget_association_status %}
-The status of the campaign's budget.
+Indicates whether or not the campaign is currently spending from the budget mentioned in the BudgetName column. The possible values are `Current` and `Ended`.
+{% enddocs %}
+
+{% docs budget_name %}
+The status of the campaign's budget. This column will be empty for unshared budgets.
```

Comment:
```suggestion
The name of the budget. This column will be empty for unshared budgets.
```
I think something just got shifted here. [Here](https://learn.microsoft.com/en-us/advertising/reporting-service/campaignperformancereportcolumn?view=bingads-13#budgetname) is the definition I was looking at!

Reply:
doh good catch! added and regenerated the docs